src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -122,6 +123,23 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 
 		this._register(this._onAddConnectionProfile);
 		this._register(this._onDeleteConnectionProfile);
+
+		// keeps azure tokens refreshed every 10 minutes
+		const seconds = 600;
```

Comment:
rename to tokenRefreshInterval

Reply:
And include a description of why you chose 10 minutes